internal/parameter/query.go

```diff
@@ -8,10 +8,12 @@ import (
 
 // AddToQuery add a key/value pair to an URL query
 func AddToQuery(query url.Values, key string, value interface{}) {
-	if value == nil {
+	elemValue := reflect.ValueOf(value)
+
+	if elemValue.Kind() == reflect.Ptr && elemValue.IsNil() {
```

Comment:
Update the condition to check `reflect.Invalid`
```Go
if elemValue.Kind() == reflect.Invalid || elemValue.Kind() == reflect.Ptr && elemValue.IsNil() {
```

Reply:
:+1: